sqlitecluster/SQLiteClusterMessenger.cpp

```diff
@@ -82,6 +82,14 @@ bool SQLiteClusterMessenger::runOnLeader(BedrockCommand& command) {
     while (chrono::steady_clock::now() < (start + 5s) && !sent) {
         string leaderAddress = _node->leaderCommandAddress();
         if (leaderAddress.empty()) {
+            // If there's no leader, it's possible we're supposed to be the leader. In this case, we can exit early.
+            auto myState = _node->getState();
+            if (myState == STCPNode::LEADING || myState == STCPNode::STANDINGUP) {
+                SINFO("[HTTPESC] I'm the leader now! Exiting early.");
```

Comment:
🤣

Reply:
makes me think of "i'm a big kid now!" commercial https://youtu.be/ztTHLDhmB58?t=25

Reply:
Hah, I was going more for this: https://www.youtube.com/watch?v=dvA-mimf2yg